### PR TITLE
Add monster choice to fill tank

### PR DESCRIPTION
**Pull request by catahnia**
Please review this, @casey-collab

### CODE REVIEW

File: greek/fill_water_tank/fill_water_tank.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 Γεμίζεις μια λεκάνη νερό κι αρχίζεις να τα ρίχνεις γρήγορα μέσα.
 
-Θα προλάβεις να τα ρίξεις όλα μέσα να λιώσουν χωρίς να σε φάνε;
\ No newline at end of file
+Θα προλάβεις να τα ρίξεις όλα μέσα να λιώσουν χωρίς να σε φάνε;
+
+[Προλαβαίνεις κι ένα ζαχαρωτό τέρας γεννιέται και βγαίνει απ τον κουβά](create_monstere/monster.md)
```
Comment (casey-collab) [inline greek/fill_water_tank/fill_water_tank.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
It looks like the file `greek/create_monster/monster.md` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by catahnia** [comment]
Please review this, @casey-collab

File: greek/fill_water_tank/fill_water_tank.md (hunk 1 above)
Comment (casey-collab) [inline greek/fill_water_tank/fill_water_tank.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**